### PR TITLE
mathext: provide the correct formula in documentation for GammaIncRegComp

### PR DESCRIPTION
**Pull request by romanwerpachowski**
Please take a look.

### CODE REVIEW

File: mathext/gamma_inc.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ func GammaIncReg(a, x float64) float64 {
 
 // GammaIncRegComp computes the complemented regularized incomplete Gamma integral.
 //  GammaIncRegComp(a,x) = 1 - GammaIncReg(a,x)
-//                    = (1/ Γ(a)) \int_0^\infty e^{-t} t^{a-1} dt
+//                    = (1/ Γ(a)) \int_x^\infty e^{-t} t^{a-1} dt
```
Comment (kortschak) [inline mathext/gamma_inc.go:25]: Can you add some spaced to make the equal signs line up (I imagine this was forgotten when the `Reg` was added.

Comment (romanwerpachowski) [reply mathext/gamma_inc.go]: Done.

**Comment by romanwerpachowski** [comment]
PTAL.

**Comment by kortschak** [comment]
Suggest "mathext: fix formula in GammaIncRegComp comment"

**Pull request merged**